packages/graalvm/src/main/kotlin/elide/runtime/intrinsics/js/err/TypeError.kt

```diff
@@ -61,12 +52,20 @@ private val TYPE_ERROR_MEMBERS_AND_PROPS = arrayOf(
  * @see AbstractJsException for the host base interface type of all JavaScript exceptions.
  * @see Error for the top-most guest-exposed base class for all JavaScript errors.
  */
-@Implementable
 public open class TypeError protected constructor (
```

Comment:
Slight issue: `TypeError` is provided by the VM. I don't know if this was already true, or what. We'll need to refactor to use the built-in instead of our own class, so I'm withholding it from installation here.